packages/opencv/config.py

```diff
@@ -35,17 +35,17 @@ def opencv(version, requires=None, default=False, url=None):
     
 package = [
     # JetPack 6
-    opencv('4.8.1', '==36.*', default=True),
+    opencv('4.10.0', '==36.*', default=True),
```

Comment:
If comfyui/flux requires opencv 4.10, can you keep 4.8 as the default, and then make add `opencv:4.10.0` to the depends list of whatever container it is that needs opencv 4.10